app/models/observation_accuracy_experiment.rb

```diff
@@ -336,15 +465,63 @@ def generate_sample
     puts "Sample generated in #{duration} seconds."
   end
 
-  def contact_validators( validator_deadline_date: 1.week.from_now.strftime( "%Y-%m-%d" ) )
+  def observation_accuracy_validator_contact( validator )
+    return false unless ( user = User.where( id: validator.user_id ).first )
+
+    admin = User.where( email: "admin@inaturalist.org" ).first
+    obs_ids = validator.observation_accuracy_samples.pluck( :observation_id )
+    num_obs = obs_ids.count
+    sample_url = FakeView.
+      identify_observations_url(
+        place_id: "any",
+        reviewed: "any",
+        quality_grade: "needs_id,research,casual",
+        id: obs_ids.join( "," )
+      )
+    experiment_url = FakeView.observation_accuracy_experiment_url( self, params: { tab: "methods" } )
+    delimited_num_obs = ApplicationController.helpers.number_with_delimiter( num_obs )
+    subject = I18n.t( :observation_accuracy_validator_email_subject2, version: version )
+    message_body = <<~HTML
```

Comment:
You previously used custom Emailer templates for emails for experiments in the past. Any reason to switch back to this non-standard approach to sending emails now?

Reply:
many people didn't notice/recieve emails for whatever reason so we're changing to using the messaging infastructure (which also generates emails)